### PR TITLE
#163445111 Admin create a political party

### PR DESCRIPTION
**Pull request by samorahno**
#### What does this PR do?
This PR gives an admin the privilege to create a Political party
#### Description of Task to be completed?
- Admin creates a political party
- Conclude test setup

#### How should this be manually tested?
By cloning the [Gith Repo](https://samorahno.github.io/Politico/UI) 
use postman to test the endpoint ```POST /parties```
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163445111
#### Screenshots (if appropriate)
image

#### Questions:
None

### CODE REVIEW

File: server/controllers/PoliticalParty.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            alias: req.body.alias,
+            hqAdress: req.body.hqAdress,
+            logoUrl: req.body.logoUrl,
+            dateCreated: `${dateObj.getFullYear()} - ${(dateObj.getMonth() + 1)} - ${dateObj.getDate()}`
+        }
+
+        politicalParties.push(party);
+      
+        res.json({
+            status: 200,
```
Comment (andela-gike) [inline server/controllers/PoliticalParty.js]: Is this the correct status code for a new creation, kindly confirm

File: server/controllers/PoliticalParty.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            data: [{
+                    "id": party.id,
+                    "name": party.name
```
Comment (andela-gike) [inline server/controllers/PoliticalParty.js]: Your response message is not descriptive enough

File: server/middleware/Validator.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    static validateString(string){
+        if(!string || string === undefined || string.toString().trim() === '' || typeof string !== 'string') return false;
+        if(string.lenght < 3 || string.lenght > 500) return false;
+        const stringAllowed = /^[a-zA-Z-'\s\d]+$/;
+        if(!string.match(stringAllowed)) return false;
+        return true;
+    }
+
+    static validateAddress(string){
+        if(!string || string === undefined || string.toString().trim() === '' || typeof string !== 'string') return false;
```
Comment (andela-gike) [inline server/middleware/Validator.js]: Confirm if you can add this if statement in one function, then call it when you want to use it.

**Review by andela-gike — CHANGES_REQUESTED** [review]

File: server/middleware/Validator.js (hunk 3 above)
Comment (samorahno) [reply server/middleware/Validator.js]: okay, i will do that

File: server/controllers/PoliticalParty.js (hunk 2 above)
Comment (samorahno) [reply server/controllers/PoliticalParty.js]: Thank you. i will make the message descriptive enough

**Review by andela-gike — APPROVED** [review]

**Pull request merged**